__tests__/OmnichannelChatSDK.spec.ts

```
@@ -898,6 +898,30 @@ describe('Omnichannel Chat SDK', () => {
             expect(chatSDK.OCClient.sessionInit).toHaveBeenCalledTimes(0);
         });
 
+        it('ChatSDK.startChat() should not call OCClient.sessionInit() if OCClient.getChatToken() fails with 401 InvalidAuthentication', async () => {
```

`ChatSDK.startChat() should fail with "InvalidAuthentication" if OCClient.getChatToken() fails with 401`

---

edited description